modules/aggregation-layers/src/heatmap-layer/heatmap-layer-utils.js

```diff
@@ -0,0 +1,113 @@
+import {experimental, createIterable} from '@deck.gl/core';
+const {count} = experimental;
+
+// Parse input data to build positions, wights arrays.
+export function parseData(data, getPosition, getWeight) {
```

Comment:
Use attribute manager?

Reply:
Thanks for the help @Pessimistress , updated with attributeManager integration.